apps/src/applab/applab.js

```diff
@@ -855,7 +853,7 @@ Applab.init = function (config) {
       localeDirection={studioApp.localeDirection()}
       editCode={!!config.level.editCode}
       readonlyWorkspace={Applab.reduxStore.getState().level.isReadOnlyWorkspace}
-      extraControlRows={extraControlRows}
+      showDebugger={showDebugButtons || showDebugConsole}
```

Comment:
Disregard: As long as we're using `JsDebugger` in the non-connected `CodeWorkspace` context, we need to do things the way you've got it now.